view_all_inc.php

```diff
@@ -223,7 +223,7 @@ function write_bug_rows( array $p_rows ) {
 <?php
 		if( $g_checkboxes_exist ) {
 			echo '<label class="inline">';
-			echo '<input class="ace check_all" type="checkbox" id="bug_arr_all" name="bug_arr_all" value="all" class="check_all input-small" />';
+			echo '<input class="ace check_all input-small" type="checkbox" id="bug_arr_all" name="bug_arr_all" value="all" />';
```

Comment:
Maybe, but as you can see I just merged 'classes'. Should it be 'input-sm' here?